tests/integration_tests/functional/test_rate_limiter.py

```diff
@@ -165,7 +165,6 @@ def test_rx_rate_limiting_cpu_load(test_microvm_with_api, network_config):
     )
 
     test_microvm.start()
-
```

Comment:
nit: since there is no change in this file, can we remove this file from the pr?

Reply:
I'd be inclined to leave it in unless there's any non-nit things that'll have to be addressed anyway